core/src/main/scala/caliban/introspection/Introspector.scala

```diff
@@ -32,8 +37,22 @@ object Introspector {
   /**
    * Generates a schema for introspecting the given type.
    */
-  def introspect(rootType: RootType): RootSchema[Any] = {
-    val types = rootType.types.updated("Boolean", Types.boolean).values.toList.sortBy(_.name.getOrElse(""))
+  def introspect[R](
+    rootType: RootType,
+    introWrappers: List[IntrospectionWrapper[R]]
```

Comment:
Can you add a default argument (Nil) here since this is part of the public interface?